.github/workflows/docker-update.yml

```diff
@@ -14,8 +14,26 @@ jobs:
         with:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
-      - name: Push to Docker Hub
+      - name: Push to Docker Hub latest
         uses: docker/build-push-action@v2
         with:
           push: true
           tags: profiq/node-java-chrome-ff:latest
+      - name: Get latest chrome version
+        run: |
+          echo ::set-env name=CHROME_VERSION::$(google-chrome --version 2>&1 | head -1 | cut -d'"' -f2 | sed '/^1\./s///' | cut -d'.' -f1 | sed '/Google Chrome /s///')
+      - name: Push to Docker Hub with chrome version
+        uses: docker/build-push-action@v2
+        with:
+          push: true
+          tags: profiq/node-java-chrome-ff:${{ env.CHROME_VERSION }}
```

Comment:
We don't need to push it twice. We should be able to push with multiple tags for single image 
https://github.com/docker/build-push-action/blob/master/docs/advanced/tags-labels.md